<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDisc</class>
 <widget class="QWidget" name="NewDisc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="DiscF" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>731</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>14</number>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="2" column="3">
     <widget class="QWidget" name="Disc7" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="Disc1" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QWidget" name="Disc6" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="Disc0" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QWidget" name="Disc3" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="Disc4" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="Disc5" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="Disc2" native="true">
      <property name="minimumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>158</width>
        <height>140</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="Disc01" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QLabel" name="labName01">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>name01</string>
       </property>
      </widget>
      <widget class="QLabel" name="labDiscAu01">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>author</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="Disc02" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QLabel" name="labDiscAu02">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>author</string>
       </property>
      </widget>
      <widget class="QLabel" name="labName02">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>name01</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="Disc03" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QLabel" name="labDiscAu03">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>author</string>
       </property>
      </widget>
      <widget class="QLabel" name="labName03">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>name01</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QWidget" name="Disc04" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QLabel" name="labDiscAu04">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>author</string>
       </property>
      </widget>
      <widget class="QLabel" name="labName04">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>name01</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
